lib/bamboo/plug/email_preview_plug.ex

```diff
@@ -10,19 +10,24 @@ defmodule Bamboo.EmailPreviewPlug do
 
   ## Using with Plug or Phoenix
 
-      # Make sure you are using Bamboo.LocalAdapter in your config
+      # Ensure you are using Bamboo.LocalAdapter in your config
```

Comment:
Wasn't sure that this should explicitly state "in your _dev_ config" give the references to `if Mix.env == :dev do` below?